src/Blocks/BlockCactus.h

```diff
@@ -1,18 +1,19 @@
 
 #pragma once
 
-#include "BlockHandler.h"
+#include "BlockPlant.h"
 
 
 
 
 
 class cBlockCactusHandler :
-	public cBlockHandler
+	public cBlockPlant
 {
+	typedef cBlockPlant Super;
 public:
 	cBlockCactusHandler(BLOCKTYPE a_BlockType)
-		: cBlockHandler(a_BlockType)
+		: cBlockPlant(a_BlockType, false)
```

Comment:
You typedeffed `Super` (which is an excellent idea, btw, do that often!), so use it here, too.